docs/content/latest/secure/authentication/ldap-authentication-ycql.md

```diff
@@ -0,0 +1,235 @@
+
+Here is an example for search + bind mode:
+
+```sh
+--use_cassandra_authentication=true --ycql_use_ldap=true --ycql_ldap_server=ldap://ldap.yugabyte.com:389 --ycql_ldap_base_dn="dc=yugabyte, dc=com" --ycql_ldap_bind_dn="cn=admin,dc=example,dc=org" --ycql_ldap_bind_passwd=admin --ycql_ldap_search_attribute=uid
+```
+
+### Configurations
+
+The configurations supported for search + bind mode.
+
+<table>
+  <tr>
+   <td><strong>Tserver gflag name</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+   <td><strong>Default value</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_base_dn
+   </td>
+   <td>Specifies the base directory to begin the user name search
+   </td>
+   <td>''
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_bind_dn
+   </td>
+   <td>Specifies the username to perform the initial search when doing search + bind authentication
+   </td>
+   <td>''
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_bind_passwd
+   </td>
+   <td>Password for username being used to perform the initial search when doing search + bind authentication
+   </td>
+   <td>''
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_search_attribute
+   </td>
+   <td>Attribute to match against the username in the search when doing search + bind authentication. If no attribute is specified, the uid attribute is used
+   </td>
+   <td>''
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_search_filter (not supported yet)
+   </td>
+   <td>The search filter to use when doing search + bind authentication
+   </td>
+   <td>''
+   </td>
+  </tr>
+</table>
+
+## Example
+
+To use LDAP password authentication on a new YugabyteDB cluster, follow these steps:
+
+1. Use tserver gflags to enable LDAP authentication on YB-TServer in simple bind mode. Use the below configuration to start a YugabyteDB cluster.
+
+    ```sh
+    --use_cassandra_authentication=true --ycql_use_ldap=true --ycql_ldap_server=ldap://ldap.forumsys.com:389 --ycql_ldap_user_prefix=uid= --ycql_ldap_user_suffix=, dc=example, dc=com --ycql_ldap_users_to_skip_csv=cassandra
+    ```
+
+    {{< note title="Note" >}}
+In the above sample configuration, we are using an [online LDAP test server](https://www.forumsys.com/tutorials/integration-how-to/ldap/online-ldap-test-server/) for setting up the LDAP authentication with YugabyteDB.
+    {{< /note >}}
+
+    The --ycql_ldap_users_to_skip_csv=cassandra gflag allows access to the user `cassandra` with password authentication. This allows the administrator to log in for setting up the roles (and permissions) for the LDAP users.
```

Comment:
nit: I think --ycql_ldap_users_to_skip_csv=cassandra would be clearer if in a code snippet (`--ycql_ldap_users_to_skip_csv=cassandra`)

Reply:
done